data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,34 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4105508"
+    name = "pr_cianorte"
+    allowed_domains = ["cianorte.pr.gov.br"]
+    start_urls = ["https://cianorte.pr.gov.br/orgao-oficial/"]
+    current_page = 1
+    base_url = "https://cianorte.pr.gov.br/orgao-oficial/pagina/"
+
+    def parse(self, response):
+        lines = response.css("tr")[1:]
```

Comment:
**suggestion**: You don't need to skip the first row, considering that the table has a well defined structure like:
```
<table>
  <thead>
    <tr><th>Table headers</th></tr>
  </thead>response.css("tr")[
  <tbody>
    <tr><td>Column you want</td></tr>
  </tbody>
</table>
```
You can use a more specific selector and get only the row inside `tbody` tag:
`lines = response.css("tbody tr")`